Ginger/GingerTest/BusinessFlowLib/BusinessFlowPOMTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿
+using Ginger;
+using Ginger.BusinessFlowWindows;
+using GingerCore;
+using GingerTest.POMs;
+using GingerTestHelper;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
+
+namespace GingerTest.BusinessFlowLib
+{
+    [TestClass]
+    [Level3]
+    public class BusinessFlowPOMTest
+    {
+       static GingerAutomator mGingerAutomator;
+        [ClassInitialize]
+        public static void ClassInit(TestContext testContext)
+        {
+            string sampleSolutionFolder = TestResources.GetTestResourcesFolder(@"Solutions"+ Path.DirectorySeparatorChar +"TestUndoBusinessFlow");
+            Console.WriteLine("POMTest Folder" + sampleSolutionFolder);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)